apps/core/tailwind.config.js

```diff
@@ -104,6 +107,17 @@ module.exports = {
       animation: {
         shimmer: 'shimmer 2s infinite linear',
       },
+      height: {
+        'verticalListShort': '21vh',
+        'verticalListLong': '57vh',
+        '4.5': '1.125rem'
+      },
+      width: {
+        '4.5': '1.125rem'
```

Comment:
You can configure this as part of the spacing scale so that it's available everywhere, see here: https://tailwindcss.com/docs/width#customizing-your-theme

As a side-note, we should work with design to rework this spacing scale because I think we're pretty inconsistent with it.